### PR TITLE
Implemented 10 second delay for splash screen

### PR DESCRIPTION
**Pull request by newcomb-luke**
'Twas requested

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -170,7 +179,7 @@ fn model(state: &Mutex<AppState>, audio: AudioManager) {
     }
 
     // Keep track of the joystick's axes
-    // This is only called if jostick is the input mode otherwise goes to keyboard state input
+    // This is only called if joystick is the input mode otherwise goes to keyboard state input
```
Comment (HernanW4) [inline src/lib.rs:182]: Thanks for fixing my typo.

**Review by HernanW4 — APPROVED** [review]

File: src/lib.rs [hunk 2]
```diff
@@ -159,6 +159,15 @@ fn model(state: &Mutex<AppState>, audio: AudioManager) {
 
     let mut last_update = Instant::now();
 
+    // In units of milliseconds. The time after the last dial alarm was acknowledged until
+    // the "Trial Complete!" splash screen is shown.
+    const SPLASH_SCREEN_DELAY: u64 = 10_000;
```
Comment (TroyNeubauer) [inline src/lib.rs]: nit: Use duration here to keep the unit with the type information
```suggestion
    const SPLASH_SCREEN_DELAY: Duration = Duration::from_secs(10);
```

Comment (newcomb-luke) [reply src/lib.rs]: Good point

Comment (newcomb-luke) [reply src/lib.rs]: Fixed

**Review by TroyNeubauer — APPROVED** [review]
Looks good!

**Pull request merged**